src/Terminal/HelpText.ts

```diff
@@ -5,7 +5,7 @@ export const TerminalHelpText: string[] = [
   "    analyze                          Get information about the current machine ",
   "    backdoor                         Install a backdoor on the current machine ",
   "    buy [-l/-a/program]              Purchase a program through the Dark Web",
-  "    cat [file]                       Display a .msg, .lit, or .txt file",
+  "    cat [file]                       Display the contents of a text file",
```

Comment:
We have many file types:
- Message files (.msg)
- Literature files (.lit)
- Text files (.txt, .json)
- Script files (.js, .jsx, .ts, .tsx, .script)

```suggestion
  "    cat [file]                       Display message files (.msg), literature files (.lit), text files (.txt, .json), and script files (.js, .jsx, .ts, .tsx, .script)",
```

Reply:
I think I've actually just say "display the contents of a file"
This help text is meant to be as brief as possible, and while there *are* some files cat won't display, they are rare.